api/bindings.h

```diff
@@ -129,15 +145,15 @@ typedef struct GoQuerier {
 
 Buffer allocate_rust(const uint8_t *ptr, uintptr_t length);
 
-AnalysisReport analyze_code(cache_t *cache, Buffer checksum, Buffer *err);
+AnalysisReport analyze_code(cache_t *cache, ByteSliceView checksum, Buffer *err);
```

Comment:
Wouldn't `BufferView` be a better name?

Reply:
ByteSliceView is not a view into a `Buffer`, but into a Go by [slice](https://blog.golang.org/slices-intro) `[] byte`.

`Buffer` is a interface type created by us that can be converted to `Vec<u8>` in Rust and `[] byte` in Go. It does much more than the simpler `ByteSliceView`.